lib/archethic_web/faucet_rate_limiter.ex

```diff
@@ -0,0 +1,126 @@
+defmodule ArchEthicWeb.FaucetRateLimiter do
+  @moduledoc false
+
+  use GenServer
+
+  @faucet_rate_limit Application.compile_env!(:archethic, :faucet_rate_limit)
+  @faucet_rate_limit_expiry Application.compile_env!(:archethic, :faucet_rate_limit_expiry)
+  @block_period_expiry @faucet_rate_limit_expiry
+  @clean_time @faucet_rate_limit_expiry
+
+  def start_link(opts) do
+    GenServer.start_link(__MODULE__, opts, name: __MODULE__)
+  end
+
+  @doc """
+  Register a faucet transaction address to monitor
+  """
+  @spec register(binary(), non_neg_integer()) :: :ok
+
+  ## Client Call backs
+  def register(address, start_time)
+      when is_binary(address) and is_integer(start_time) do
+    GenServer.cast(__MODULE__, {:register, address, start_time})
+  end
+
+  def reset() do
```

Comment:
When this is used ?

Reply:
I used this while testing, to reset .... Removed now